zc_install/includes/languages/en_us/main.php

```diff
@@ -252,19 +253,19 @@
 'TEXT_HELP_TITLE_DBPASSWORD' => 'Database Password',
 'TEXT_HELP_CONTENT_DBPASSWORD' => 'What is the password assigned to the MySQL username you created for this database.',
 'TEXT_HELP_TITLE_DBNAME' => 'Database Name',
-'TEXT_HELP_CONTENT_DBNAME' => 'What is the name of the database used to hold the data? An example database name is "zencart" or "myaccount_zencart".<br>NOTE: You must create this database BEFORE you can proceed with Zen Cart&reg; installation here.<br>You can create your MySQL database using your hosting company\'s control panel.',
+'TEXT_HELP_CONTENT_DBNAME' => 'What is the name of the database used to hold the data?<br>An example database name is "zencart" or "myaccount_zencart".<br>NOTE: <strong>You</strong> must create this database BEFORE you can proceed with the Zen Cart&reg; installation.<br>You can create your MySQL database using your hosting company\'s control panel.',
 'TEXT_HELP_TITLE_DEMODATA' => 'TEXT_DATABASE_SETUP_LOAD_DEMO',
-'TEXT_HELP_CONTENT_DEMODATA' => 'If you choose to load Demo Data, we will install a base set of products and categories, with sales and specials and attributes and more. These are useful for you to play around and see how various combinations can be set up and how they can look on your storefront.<br><br>You can certainly delete the demo products (by hand) later, or once you\'ve toyed with the samples, you can re-run this install and choose to not install the demo data, and thus have a fully clean site for setting up your own new store.',
+'TEXT_HELP_CONTENT_DEMODATA' => 'Demo Data sets up some example categories and products, with a range of product attributes (variants), discounts, sales, special offers etc. These are useful for new users to investigate the capabilities of the shop and how these product options are displayed.<br><br>Subsequently you may delete the demo products manually, or you can re-run this Clean Install process without the Demo Data to start afresh with your own products.',
 'TEXT_HELP_TITLE_DBCHARSET' => 'Database Character Set',
 'TEXT_HELP_CONTENT_DBCHARSET' => 'Most stores will use utf8mb4 or utf8.<br>If you don\'t have a reason to use something else, use utf8mb4.',
 'TEXT_HELP_TITLE_DBPREFIX' => 'Database Tablename Prefix',
-'TEXT_HELP_CONTENT_DBPREFIX' => 'What is the prefix you would like used for database tables?  Example: <strong>zen_</strong><br><strong class="alert">TIP: Leave empty if no prefix is needed.</strong><br />You can use prefixes to allow more than one store to share the same database.',
+'TEXT_HELP_CONTENT_DBPREFIX' => 'What prefix would you like to use for the Zen Cart&reg; database tables?<br>Historically it was common on shared hosting to have a very limited number of databases offered. Consequently several applications may have shared a single database, using table prefixes to maintain the data separate.<br>Example: <strong>zen_</strong><br><strong class="alert">TIP: Leave empty if no prefix is needed.</strong>',
```

Comment:
Is the history lesson still needed?

Is `to maintain the data separate.` good grammar?

Reply:
I think the history lession is of use. Maybe you don't remember that feeling of total ignorance that a new user has. Prefixes may be ancient history now, but the option is there and anything that reduces the "I don't understand anything" feeling is positive. I think.
I try.